apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -172,67 +167,98 @@ function TransferCoinPage() {
         ({ suiObjects }) => suiObjects.loading && !suiObjects.lastSync
     );
 
+    const handleNextStep = useCallback(
+        (
+            formData: FormValues,
+            { setSubmitting }: FormikHelpers<FormValues>
+        ) => {
+            setCurrentStep((prev) => prev + 1);
+            setFormData(formData);
+            setSubmitting(false);
+        },
+        []
+    );
+
     if (!coinType) {
         return <Navigate to="/" replace={true} />;
     }
 
     const StepOneForm = (
-        <Formik
-            initialValues={formData}
-            validateOnMount={true}
-            validationSchema={validationSchemaStepOne}
-            onSubmit={handleNextStep}
-        >
-            <StepOne
-                coinSymbol={coinSymbol}
-                coinType={coinType}
-                onClearSubmitError={handleOnClearSubmitError}
-                onAmountChanged={(anAmount) => setAmountToSend(anAmount)}
-            />
-        </Formik>
-    );
+        <div className="flex flex-col w-full mt-2.5">
```

Comment:
Can we move all this to the `StepOne` component and rename it to `SendTokenForm` or something better and use props for what data, submit etc. So that component can handle everything related with user input stuff